webui/src/pages/auth/groups/group/members.jsx

```diff
@@ -36,14 +36,29 @@ const GroupMemberList = ({ groupId, after, onPaginate }) => {
     }, [refresh]);
 
 
-    const searchUsers = async (prefix, maxResults, resolveUserDisplayNameFN = (user => user.id)) => {
+    const searchUsers = async (prefix, after, resolveUserDisplayNameFN = (user => user.id)) => {
         let allUsersList = allUsers;
         if (allUsersList.length === 0) {
             allUsersList = await allUsersFromLakeFS(resolveUserDisplayNameFN)
             setAllUsers(allUsersList)
         }
         let filteredUsers = allUsersList.filter(user => resolveUserDisplayNameFN(user).startsWith(prefix));
-        return filteredUsers.slice(0, maxResults);
+        const pageSize = 5;
```

Comment:
Please make this `pageSize = 5`, that repeats in all these files, a const (wherever you see fit. I suggest `constants.ts`).